patches/inox-patchset-0009-disable-google-ipv6-probes.patch

```diff
@@ -0,0 +1,16 @@
+--- a/net/dns/host_resolver_impl.cc
```

Comment:
i think we prefer that this patch file follow the naming convention of the other patches in this repo. you can autogenerate patches by checking out https://github.com/brave/brave-browser, doing the changes in `src/`, then `npm run update_patches` to update patches in `src/brave`

Reply:
is there another way to block this? I believe some of the others use the network delegate? cc @pilgrim-brave

Reply:
I guess it's a DNS request so the network delegate wouldn't apply, but just want to make sure that there isn't an alternative to patching

Reply:
maybe patch DefaultClientSocketFactory to return our own subclass of UDPClientSocket so we can create a more extensible blocking mechanism?

Reply:
actually wouldn't even have to patch, just do a chromium_src override to replace DatagramClientSocket with BraveDatagramClientSocket

Reply:
@bridiver would it suffice if I just override `HostResolverImpl::IsIPv6Reachable` at `net/dns/host_resolver_impl.cc`? (And this assumes that the implementation won't change over time.)

This is to make the change to be as local as possible, as I don't know how to override the constant from `DatagramClientSocket`/`UDPClientSocket`, which is buried down in `HostResolver` (I haven't looked at the location where `HostResolver` is initialized).

Reply:
methods like that aren't easily overridable. It's actually quite simple to override override the socket class with a chromium_src override for `net/socket/client_socket_factory.cc`. See `chromium_src/chrome/browser/io_thread.cc` for an example